tools/cert-sidecar/app/ServiceStateMgmt.js

```diff
@@ -0,0 +1,67 @@
+const {
+  ServiceStateManager: { Manager },
+} = require('@dojot/microservice-sdk');
+
+
+/**
+ * This class has responsible for the health check and graceful shutdown
+ */
+class ServiceStateMgmt {
```

Comment:
I think it's fine to do this by now, since we don't have the addHealthChecker in the master thread, but IMO I think it would be nice to add this functionality in the SDK and then remove this and use the Manager class as it is.